app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,87 @@
+class TweetsController < ApplicationController
+
+#   configure do
+#    set :public_folder, 'public'
+#    set :views, 'app/views'
+#    enable :sessions
+#   #   # set :session_secret, ENV.fetch('SESSION_SECRET') { SecureRandom.hex(64) }
+#   #   # set :session_secret, SecureRandom.hex(64)
+# end
+
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'/tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if params.value?('')
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.